### PR TITLE
Update ecip-1066.md

### PR DESCRIPTION
**Pull request by chris-mercer**
Add Spiral

### CODE REVIEW

File: _specs/ecip-1066.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Ethereum Classic network mainnet.
...
+| Spiral <sub>Shanghai</sub>                   | 19250000   | 2024-02-04 | [EIP-3651](https://eips.ethereum.org/EIPS/eip-3651) <br/> [EIP-3855](https://eips.ethereum.org/EIPS/eip-3855) <br/> [EIP-3860](https://eips.ethereum.org/EIPS/eip-3860) <br/> [EIP-6049](https://eips.ethereum.org/EIPS/eip-6049)                                                                                                                                                                                                                                                                                                                                                                    | [ECIP-1109](https://ecips.ethereumclassic.org/ECIPs/ecip-1109) |
```
Comment (ziogaschr) [inline _specs/ecip-1066.md:16]: Just adding an extra column for blog to be consistent with rest entries.

```suggestion
| Spiral <sub>Shanghai</sub>                   | 19250000   | 2024-02-04 | [EIP-3651](https://eips.ethereum.org/EIPS/eip-3651) <br/> [EIP-3855](https://eips.ethereum.org/EIPS/eip-3855) <br/> [EIP-3860](https://eips.ethereum.org/EIPS/eip-3860) <br/> [EIP-6049](https://eips.ethereum.org/EIPS/eip-6049)                                                                                                                                                                                                                                                                                                                                                                    | [ECIP-1109](https://ecips.ethereumclassic.org/ECIPs/ecip-1109) | |
```

**Comment by meowsbits** [comment]
I would like more descriptive commits and branch name (@chris-mercer especially since you're pushing directly to the ethereumclassic remote). Without viewing the patch itself I have no idea what's going on here.

**Comment by gitr0n1n** [comment]
> I would like more descriptive commits and branch name (@chris-mercer especially since you're pushing directly to the ethereumclassic remote). Without viewing the patch itself I have no idea what's going on here.

Done! [7c210cf6e166e30992f8a6f77ec5f36289784e47](https://github.com/ethereumclassic/ECIPs/pull/526/commits/7c210cf6e166e30992f8a6f77ec5f36289784e47)

**Review by meowsbits — APPROVED** [review]

**Comment by realcodywburns** [comment]
How was eip-6049 implemented? It is not included in the 'expected action' portion and the spec is dubious + poorly defined. My understanding is this is a warning in documentation(?) to not use selfdestruct. As written, 6049 opens the door to reuse the opcode for something that a contract may not have intended when deployed. I would recommend having it as a permanent 'noop' and not be allowed to be repurposed. Its implementation or lack there of is included in this ecip update

**Comment by meowsbits** [comment]
In core-geth `IsEIP6049` is inoperative; it's just a paper placeholder, and afaiu that's the spec.  Duly noted on the suggestion to permanent noop it.

**Comment by realcodywburns** [comment]
Sounds good to me. can we squash merge this now?

**Review by meowsbits — APPROVED** [review]

**Pull request merged**